Providers/Modules/Plugins/Common/plugin/out_oms_diag.rb

```diff
@@ -0,0 +1,111 @@
+module Fluent
+
+  class OutputOMSDiag < BufferedOutput
+
+    Plugin.register_output('out_oms_diag', self)
```

Comment:
@lagalbra I would prefer that diagnostic logging be available from this release onwards without modifying the omsagent.conf. I would make a change in oms_diag_lib to drop the log if the version is below the intended release version, so that in earlier versions there is no diagnostic logging emitted from any of input plugins (updated via DSC to utilize diagnostic logging).
However if it is intended to have backward compatibility then I have the code ready as part of this PR but the testing and verification on many versions will be a big effort.